content/applications/productivity/discuss/overview/plan_activities.rst

```diff
@@ -1,64 +1,198 @@
-====================================
-Get Organized by Planning Activities
-====================================
+==========
+Activities
+==========
 
-When you plan activities you minimize the risk of uncertainties, as you provide clear directions
-for the course of your next action. In addition to that, you do not leave space for wasteful
-activities and reduce the chance of having overlapping actions between team members.
+*Activities* are follow-up tasks tied to a record in an Odoo database. Activities can be scheduled
+on any page of the database that contains a chatter thread.
 
-Where do I see my schedule activities?
-======================================
+Schedule activities
+===================
 
-Access and manage your activities wherever you are in Odoo by the *Activities* menu.
+Activities are created on a record through the *chatter*. To create an activity on a *CRM* lead, go
+to the :menuselection:`Pipeline` and click on a :guilabel:`lead`.
 
-.. image:: plan_activities/activities_menu.png
+Above the chatter, click :guilabel:`Schedule activity`. On the popup window, select an
+:guilabel:`Activity Type` from the drop down.
+
+.. note::
+   The :guilabel:`Due Date` will auto-populate based on the configuration settings for the selected
+   activity type. This date can be changed by selecting a day on the calendar in the :guilabel:`Due
+   Date` field.
+
+Enter a title for the activity in the :guilabel:`Summary` field. To assign the activity to a
+different user, select their name from the :guilabel:`Assigned to` drop-down. Otherwise, the user
+creating the activity will be automatically assigned. Add any additional information in the
+:guilabel:`Log a note...` field.
+
+Click :guilabel:`Schedule`.
+
+.. image:: plan_activities/schedule-pop-up.png
    :align: center
-   :height: 300
-   :alt: View of crm leads page emphasizing the activities menu for Odoo Discuss
+   :alt: View of CRM leads and the option to schedule an activity.
 
-Plan activities
-===============
+.. note::
+   Depending on the activity type, the :guilabel:`Schedule` button may be replaced by a
+   :guilabel:`Save` button.
 
-Activities can be planned and managed from the chatter by clicking on *Schedule activity*,
-or through Kanban views.
+Scheduled activities are added to the chatter for the record under :guilabel:`Planned activities`.
 
-.. image:: plan_activities/schedule_activity.png
+.. image:: plan_activities/chatter-activities.png
    :align: center
-   :height: 300
-   :alt: View of crm leads and the option to schedule an activity for Odoo Discuss
+   :alt: View of CRM leads and the option to schedule an activity.
```

Comment:
This highlight is confusing, we are talking about the newly posted activity and "Schedule Activity" is highlighted.